lib/src/app/app_config.dart

```diff
@@ -0,0 +1,15 @@
+abstract class AppConfig {
+  final String baseUrl;
+
+  AppConfig(this.baseUrl);
+}
+
+class DevAppConfig implements AppConfig {
```

Comment:
Rozumiem ideę, ale póki jest jak jest, czyli configi się nie różnią to zostawiłbym jeden, później jak się będą różnić faktycznie to się doda różne implementacje ;)